cogs/userinfo.py

```diff
@@ -0,0 +1,35 @@
+from discord.ext import commands
+import discord
+
+class useinfo(commands.Cog, name='Userinfo'):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(description = 'Information about a person.')
+    async def userinfo(self, ctx, member: discord.Member = None):
+        if not member:
```

Comment:
You can use `member = member or ctx.author` so you won’t need the `if not member` and you will only need the embed creating code once